src/simulation/LoadGenerator.h

```diff
@@ -160,10 +199,54 @@ class LoadGenerator
 
     uint32_t mWaitTillCompleteForLedgers{0};
 
-    void reset();
+    // The following maps store state information for the SOROBAN_INVOKE mode.
+    // For all other loadgen modes, there are no dependent TXs after creating
+    // source accounts. However, for SOROBAN_INVOKE, we must deploy WASM blobs,
+    // create contract instances, and create storage entries before invoking the
```

Comment:
If we just have one Wasm entry, we only load the Wasm once, even though every TX is charged read resources for the entry. For the worst case scenario, every TX would be loading a different Wasm blob such that we have no cache hits. My thinking is that each Wasm blob has the same source code, but we add padding so that the size and hash varies, creating more IO work. This might be overkill, we could also just load more contract data and probably achieve something similar...

Reply:
Sorry, I probably was unclear. The confusion point to me is why storage entry setup is mentioned alongside Wasm/contract instances. It should be possible to invoke contract without setting up any additional entries, right?